packages/web-client/tests/unit/layer-1-connection-manager-test.ts

```diff
@@ -0,0 +1,253 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { ConnectionManager } from '@cardstack/web-client/utils/web3-strategies/layer-1-connection-manager';
+import Web3 from 'web3';
+import Sinon from 'sinon';
+
+module('Unit | layer 1 connection manager', function (hooks) {
+  setupTest(hooks);
+
+  test('It should be able to be instantiated for Kovan and Sokol', async function (assert) {
+    let sokolConnectionManager = new ConnectionManager('sokol');
+    let kovanConnectionManager = new ConnectionManager('kovan');
+
+    assert.ok(
+      sokolConnectionManager,
+      'Sokol connection manager is instantiated'
+    );
+
+    assert.ok(
+      kovanConnectionManager,
+      'Kovan connection manager is instantiated'
+    );
+  });
```

Comment:
With these kinds of assertions, I’d expect at least one more that exercises the opposite case, presumably that it can’t be instantiated for some other string.

Reply:
Wait, I just realised I made a mistake and Sokol should not be allowed to be instantiated. Thanks for the reminder, I need to tighten the typing on this (and the layer 1 chain).

But otherwise, I would have been relying on Typescript to ensure that only certain strings (valid layer 1 networks) could be used to instantiate this.

Reply:
Is Typescript enough, or should I also add assertions within the constructor to prevent this?

Reply:
I feel like Typescript is enough, and there probably is no necessity for a test for successful instantiation in that case, so this test was removed.

Reply:
agreed! 🤓